debian/rules

```diff
@@ -213,6 +213,18 @@ override_dh_auto_install:
 	cp -ar factory/* debian/tmp
 	mkdir -p debian/tmp/usr/bin
 	mkdir -p debian/tmp/usr/lib64
+
+	# splash functionality
+	mkdir -p debian/tmp/usr/share/plymouth/themes/
+	cp -a vendor/plymouth-theme-ubuntu-core/ubuntu-core \
+		debian/tmp/usr/share/plymouth/themes/
+	mkdir -p debian/tmp/usr/share/fonts/
+	cp /usr/share/fonts/truetype/ubuntu/Ubuntu-R.ttf \
+		debian/tmp/usr/share/fonts/Plymouth.ttf
+	# We need the seat rules for drm
+	sed 's#{{ROOTBINDIR}}#/bin#' vendor/systemd/src/login/71-seat.rules.in > \
```

Comment:
This is incomplete. I get errors printed for lines 59 and 63.

Reply:
Nice catch, I've fixed now